internal/remote/firecrest/controller.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2025.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
```

Comment:
May I suggest the adoption of the SPDX standard ?
It's more compact and there are several other OpenSource projects that uses it such as KDE and Qt.
It would look something like:

```suggestion
/*
  SPDX-FileCopyrightText: Copyright © 2025 SDSC <info@sdsc.ch>
  SPDX-License-Identifier: Apache-2.0
*/
```

Reply:
Can you please make an issue for this? We won't solve this with a comment on a stacked PR. This will be mostly down to some legal discussion.